test('hello world', () => {

})

test('this fails', () => {
    throw new Error('failure');
})